testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts

```diff
@@ -0,0 +1,13 @@
+function runCode() {
```

Comment:
Is it necessary to create a new test project just for this test? Could we re-use one of the existing ones?

Reply:
What disadvantage do you see in this test having it's own test project?

Reply:
Just excess bloat of the testdata/ dir, and the repo in general. This test really only cares about the launch.json params, and whether or not we can hit that url, so the test project itself is superfluous. We already have a bunch of little test projects that would meet the same need.

Why add more stuff to maintain when we don't need to?

Reply:
I'm okay with trying to reuse projects. The existing ones have very specific behaviors that we want for the individual tests, so I don't see a great candidate to reuse it. I can move this project to a reusableProjects\ folder so people know this project doesn't have any particular behaviors, and they can reuse it for future tests. What do you think?

Reply:
Changed to use simple project as discussed offline